modules/ROOT/pages/indexes-for-full-text-search.adoc

```diff
@@ -222,10 +224,18 @@ This means that updates will be applied in a background thread "as soon as possi
 .+CREATE FULLTEXT INDEX+
 ======
 
+.Parameters
+[source,javascript, indent=0]
+----
+{
+  "name": "taggedByRelationshipIndex"
+}
+----
+
```

Comment:
Should the only create full-text index be with using a param?
Couldn't we keep this example as is, and then say something like. 
"As of Neo4j 5.16, it is also possible to create a named full-text index using parameters."  and then have an example of this?

Reply:
We have the node example which doesn't use parameter, so we do have one of each. It didn't feel like it matched the page outline to add another example like how it's done for the range/text/point

Reply:
Ok. Maybe at the start of the example box, we can say something like
"This statement creates a named full-text index using a parameter. Creating and dropping indexes using parameters was introduced in Neo4j 5.16."

And then I can see if I can make it fit in a more neater way in my new full-text index page.